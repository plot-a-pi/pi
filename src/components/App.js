import React from 'react';
import Footer from './common/Footer';

export default function App() {
  return (
    <h1>Hello World</h1>
  );
}
