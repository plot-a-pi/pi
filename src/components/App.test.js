import React from 'react';
import { mount } from 'enzyme';
import App from './App';
import { SocketProvider } from 'react-socket-io-hooks';

describe('App component', () => {
  it('renders App', () => {
    const wrapper = mount(
      <SocketProvider>
        <App />
      </SocketProvider>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
