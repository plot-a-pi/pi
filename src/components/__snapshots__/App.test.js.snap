// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders App 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    path="/teachers"
  />
  <Route
    component={[Function]}
    path="/teacher-sessions"
  />
  <Route
    component={[Function]}
    path="/session/:id"
  />
  <Route
    component={[Function]}
    path="/submit-to-global"
  />
  <Route
    path="/session-graph/:id"
    render={[Function]}
  />
</Switch>
`;
